_includes/tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/create-topic.sh

```diff
@@ -0,0 +1 @@
+docker exec -i broker /usr/bin/kafka-topics --create --topic example-topic --bootstrap-server broker:9092 --replication-factor 1 --partitions 1
```

Comment:
here you use `docker exec`, but below you use `docker-compose exec`.  IMO stick with `docker-compose exec` throughout.  One benefit of the `docker-compose exec` is that the 'interactive' flag is implied, so you wont need that like you do with docker exec

Reply:
ack, good suggestion @rspurgeon.